app/models/form_profiles/va_0873.rb

```diff
@@ -64,24 +63,40 @@ def initialize_ava_profile
 
     VA0873::FormAvaProfile.new(payload)
   rescue => e
-    log_exception_to_sentry(e, {}, prefill: :personal_information)
-    {}
+    handle_exception(e, :ava_profile)
   end
 
+  # Retrieves the preferred name for the user
   def preferred_name
     VAProfile::Demographics::Service.new(user).get_demographics.demographics.preferred_name.text
   rescue => e
-    log_exception_to_sentry(e, {}, prefill: :personal_information)
-    {}
+    handle_exception(e, :preferred_name)
   end
 
+  # Retrieves the profile from the AskVAApi service
   def profile
-    AskVAApi::Profile::Retriever.new(icn: user.icn).call
+    @profile ||= AskVAApi::Profile::Retriever.new(icn: user.icn).call
   rescue => e
-    log_exception_to_sentry(e, {}, prefill: :personal_information)
+    handle_exception(e, :profile)
```

Comment:
Any potential for PII being logged to Sentry?

Reply:
None, it'll just log the error. No PII will be in the error message.